include/exec/system_context.hpp

```diff
@@ -220,44 +221,48 @@ namespace exec {
       stdexec::set_error_t(std::exception_ptr)>;
 
     /// Implementation detail. Constructs the sender to wrap `__impl`.
-    explicit system_sender(__detail::__system_scheduler_ptr __impl)
-      : __scheduler_{std::move(__impl)} {
+    explicit __parallel_sender(__detail::__backend_ptr __impl)
+      : __scheduler_{__impl} {
     }
 
     /// Gets the environment of this sender.
     [[nodiscard]]
-    auto get_env() const noexcept -> __detail::__system_scheduler_env {
+    auto get_env() const noexcept -> __detail::__parallel_scheduler_env {
       return {__scheduler_};
     }
 
+    /// Value completion happens on the parallel scheduler.
+    parallel_scheduler
+      query(stdexec::get_completion_scheduler_t<stdexec::set_value_t>) const noexcept;
```

Comment:
i recently moved to trailing return types for all of stdexec. if you have it, `clang-tidy --fix --fix-errors <file>` will apply the changes for you.

Reply:
I did apply clang-tidy to these source files, but it also changed some other files. I only submitted changes to the "system_context"-related source files.